test/ExpressionFixtures/base.ts

```diff
@@ -811,4 +811,19 @@ line`,
 		tests: [],
 	},
 	{ expression: '={{ `\ntesting\ntesting\n` }}', tests: [] },
+	{ expression: '={{ new Date }}', tests: [] },
+	{ expression: '={{ new Date() }}', tests: [] },
+	{ expression: '={{ new Date.toISOString() }}', tests: [] },
+	{ expression: '={{ new Date().toISOString() }}', tests: [] },
+	{ expression: '={{ global.test = 3 }}', tests: [] },
+	{ expression: '={{ window.test = 3 }}', tests: [] },
+	{ expression: '={{ this.test = 3 }}', tests: [] },
```

Comment:
What are `tests` for? I don't see them being used?

Reply:
These are a hold over from when they were used in the `workflow` package. I'm actually using them in the other PR I submitted earlier today.